src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1263,6 +1263,14 @@ MulticopterPositionControl::task_main()
 				control_auto(dt);
 			}
 
+			/* weather-vane mode for vtol: disable yaw control */
+			if(_mode_auto && _pos_sp_triplet.current.disable_mc_yaw_control == true) {
```

Comment:
we should change `_mode_auto` to `!_control_mode.flag_control_manual_enabled` so this also works in offboard and it's consistent with the other checks